opm/autodiff/ThreadHandle.hpp

```diff
@@ -150,8 +150,6 @@ namespace Opm
         if( createThread )
```

Comment:
Isn't this already the case in master? The `threadObjectQueue_` is declared before `thread_ ` [here](/OPM/opm-simulators/blob/138eb2c91b05f65bb4589f8745bc8cb66fcfa31e/opm/autodiff/ThreadHandle.hpp#L136-L137)